terraform/spec/fixtures/projects/provider_with_multiple_local_path_modules/.terraform.lock.hcl

```diff
@@ -0,0 +1,20 @@
+# This file is maintained automatically by "terraform init".
+# Manual edits may be lost in future updates.
+
+provider "registry.terraform.io/mongey/confluentcloud" {
+  version     = "0.0.6"
+  constraints = "0.0.6"
```

Comment:
You might run into regressions here if a new version is released
Adding version constraints solves this https://github.com/dependabot/dependabot-core/pull/3937/files#diff-3c5654bd9cc5ac3a940be7e9806a42b2378031143810aa1f632d53492ef76763R678

Reply:
Oh will check this, i was thinking this would be fine as the constraint forces an exact version